ivy_tests/test_ivy/test_frontends/test_torch/test_other_operations.py

```diff
@@ -0,0 +1,30 @@
+# global
+from hypothesis import strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# as_dtype
```

Comment:
Would you change the comment name to clone

Reply:
Noted